src/components/Navbar.jsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Link, NavLink } from "react-router-dom";
 import "../assests/styles/navbar.css";
-import { FaSearch } from "react-icons/fa";
+// import { FaSearch } from "react-icons/fa";
```

Comment:
Remove if not required

Reply:
Yes I will be working on search in next PR